app_confetti/config.py

```diff
@@ -1,21 +1,26 @@
+import dataclasses
+
+from app_confetti import util
+
+
+@dataclasses.dataclass(frozen=True)
 class BaseConfig:
     """Configuration class.
 
-    The `environ_config` library allows for its attributes to be retrieved from
-    environment variables, the prefix for this class and the attribute name
-    defines the variable name to be used, e.g. PREFIX_SENTRY_DSN for SENTRY_DSN.
+    The `env` util function allows for attributes to be retrieved from environment variables.
 
-    This class is generally used in settings.__init__.py to instantiate an object from
-    the environment.
-
-    Inheriting classes must implement:
-        cls.LOGGING_LEVEL
-        cls.SENTRY_DSN
-        cls.ENV
+    Inheriting classes can override:
+        logging_level: str
+        sentry_dsn: str
+        env: str
     """
 
+    logging_level: str = util.env("LOGGING_LEVEL:INFO")
+    sentry_dsn: int = util.env("SENTRY_DSN:None", util.str_to_literal)
+    env: int = util.env("ENV:None", util.str_to_literal)
```

Comment:
Using dataclasses and `env` now means you must explicitly state the entire var to pull from if you want to keep using `MT4_*`

Also: `env: int`?

Reply:
mistype :)

Reply:
env: int is a mistype